Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml

```diff
@@ -61,8 +61,8 @@
       <soundHarvesting>Harvest_Standard</soundHarvesting>
       <soundHarvestFinish>Harvest_Standard_Finish</soundHarvestFinish>
       <sowWork>150</sowWork>
-      <!--<fertilityMin>0.5</fertilityMin>
-      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>-->
+      <fertilityMin>0.5</fertilityMin>
+      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>
```

Comment:
It is improved, grass does grow, but there is a lot of dirt, and yes this is with 1.12.